src/cityreader/cityreader.py

```diff
@@ -17,17 +28,27 @@
 cities = []
 
 def cityreader(cities=[]):
-  # TODO Implement the functionality to read from the 'cities.csv' file
-  # For each city record, create a new City instance and add it to the 
-  # `cities` list
-    
+    with open("cities.csv") as f:
+        csv_reader = csv.reader(f)
+        line = 0
+        for row in csv_reader:
+            if line == 0:
+                line += 1
+            else:
```

Comment:
You might be able to replace this conditional with next()
next(csv_reader) => jump one iteration in the for loop